### DISCUSSION TITLE
After new install no link with database

### DISCUSSION THREAD

**Discussion by lazycan** [YOURLS/YOURLS#3202] [Q&A] [answered]
Hello,
I had to install Yourl on a new host and it doesn't have all the old shortcuts. I changed the name of the database in the config.php file. Does anyone know why I can't find all the history?
thanks

**Comment by dgw** [comment, marked as answer]
Assuming you actually migrated the database to the new host, make sure that not just the database name matches, but also the table prefix.

If you _didn't_ export the database from the old host and import it at the new one, well… do so (_and_ make sure the config matches).

**Reply by lazycan** [reply]
Thank you dgw, it's the right solution : the 'YOURLS_DB_PREFIX' was wrong.
Thank you very much
